src/resources/armies/moloch.json

```diff
@@ -0,0 +1,332 @@
+{
+  "id": "moloch",
+  "name": "Moloch",
+  "tokens": [
+    {
+      "id": "hq",
+      "hq": true,
+      "initiative": 0,
+      "attacks": [
+        {
+          "type": "riffle",
+          "angle": -1
+        }
+      ],
+      "modifiers": [
+        {
+          "type": "strengthr",
+          "angle": -1,
+          "value": 1,
+          "self": true
+        }
+      ]
+    }, {
+      "id": "blocker",
+      "count": 2,
+      "initiative": 0,
+      "health": 3,
+      "shields": [
+       0 ]
+    }, {
+      "id": "cyborg",
+      "count": 2,
+      "initiative": 3,
+      "attacks": [
+        {
+          "type": "riffle"
+        }
+      ]
+    }, {
+      "id": "gauss_cannon",
+      "initiative": 1,
+      "health": 2,
+      "attacks": [
+        {
+          "type": "cannon",
+          "damage": 1
+        }
+      ],
+      "modifiers": [
+	{
+	  "type": "nostrength",
```

Comment:
What's this? :P

Reply:
As per conversation attack type would be enough to cover this modifier, removed